src/models/relative.js

```diff
@@ -26,29 +30,51 @@ export const isLiving = attributes => (
   attributes.IsDeceased && attributes.IsDeceased.value === 'No'
 )
 
-export const requireCitizenshipDocumentation = (attributes) => {
-  const bornInUS = attributes.Birthplace
-    && attributes.Birthplace.country
-    && countryString(attributes.Birthplace.country) === 'United States'
-
-  return !!(isCitizen(attributes)
-    && attributes.Birthplace
-    && attributes.Birthplace.country
-    && !bornInUS)
-}
+export const livesInUS = attributes => !!(
+  attributes.Address
+    && attributes.Address.country
+    && countryString(attributes.Address.country) === 'United States'
+)
 
 export const isLivingNonCitizen = attributes => (
   isLiving(attributes) && !isCitizen(attributes)
 )
+export const wasBornInUS = attributes => (
+  attributes.Birthplace
+  && attributes.Birthplace.country
+  && countryString(attributes.Birthplace.country) === 'United States'
+)
 
-export const livesInUS = attributes => !!(
-  attributes.Address
-    && attributes.Address.country
-    && countryString(attributes.Address.country) === 'United States'
+export const hasApoAddress = attributes => (
+  attributes.AlternateAddress
+  && attributes.AlternateAddress.HasDifferentAddress
+  && attributes.AlternateAddress.HasDifferentAddress === 'Yes'
+)
+
+export const requireCitizenshipDocumentation = (attributes, options) => {
+  if (options.requireRelationshipRelativesForeignBornDoc) {
+    return !!((
+      isCitizen(attributes)
+      && !wasBornInUS(attributes)
+      && !isLiving(attributes)
```

Comment:
is this correct - the documentation is required even if the relative is deceased?

Reply:
I believe so. That's how the form tracking doc reads and 18.11 in the validation matrix.